table/server/underdb/glue/src/main/java/alluxio/table/under/glue/GlueDatabase.java

```diff
@@ -316,26 +320,33 @@ public UdbTable getTable(String tableName) throws IOException {
 
       // Glue does not provide column statistic information
       List<ColumnStatisticsInfo> columnStatisticsData = new ArrayList<>();
```

Comment:
Instead of creating a new object, we can just use `Collections.emptyList()`.

Reply:
Changed.